src/Process/Pool/Logger.php

```diff
@@ -49,7 +49,13 @@ public function log($level, $message, array $context = [])
                 $logMessage->setProcessId($processId);
                 $logMessage->setProcessPath($this->_getProcess()->getPath());
                 $logMessage->setMessage($message);
-                fwrite(STDOUT, $this->getLogFormatter()->getFormattedMessage($logMessage) . "\n");
+                if (count($context) === 1) {
```

Comment:
I'm curious what the point of having `$context` be an `array` is if its only use is when there's a single element in it. Any reason to not iterate over all elements of `$context` or perhaps to throw an exception if `count($context) !== 1`?

Reply:
`\Psr\Log\LoggerInterface`